docs/usage.rst

```diff
@@ -273,3 +274,53 @@ Simply replace the ``Simulator`` with the one from NengoFPGA:
 
    with nengo_fpga.Simulator(model) as sim:
       sim.run(1)
+
+
+Maximum Model Size
+==================
+
+When running Nengo models on other hardware there is no set limit to model or
+network size. The system will continue to allocate resources (like memory) until
+it runs out which leads to different limits depending on the capabilities of
+your hardware. On the other hand, the NengoFPGA design is fixed and therefore we
+must provision resources up front. As a result, we have specific upper bounds
+which are chosen such that the resource allocation balances performance and
+flexibility for the given architecture. We store all  neuron parameters on-chip
+giving us bounds based on specific memory requirements:
+
+   - The maximum number of neurons, *N*, used to allocate memory for things like
+     neuron activity and bias.
+   - The maximum number of representational dimensions (input or output), *D*,
+     used to allocate memory for things like the input and output vector.
+   - The maximum product of neurons and dimensions, *NxD*, used to allocate
+     memory for things like encoder and decoder matrices.
+
+These maximum model size values are summarized in the table below for all
+supported hardware:
+
+.. list-table::
+   :align: center
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Device
+     - Neurons (N)
+     - Dimensions (D)
+     - N x D
+
+   * - DE1-SoC
```

Comment:
We should put these in their respective docs, and then link to them from here. That way, if we ever update the (respective) boards, we dont have to update all of the docs.

Reply:
Good idea